focus-android/app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -228,6 +239,7 @@ class BrowserToolbarIntegration(
             setBrowserActionButtons()
             observeEraseCfr()
         }
+        observeCookieBannerCfr()
```

Comment:
Should we only observe when the setting is off?

Reply:
Done